java/test/src/main/java/org/ray/api/benchmark/MicroBenchmarks.java

```diff
@@ -0,0 +1,48 @@
+package org.ray.api.benchmark;
+
+import org.ray.api.Ray;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MicroBenchmarks {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MicroBenchmarks.class);
+
+  public static Object simpleFunction() {
```

Comment:
Since this is a remote function, why there is no `@RayRemote`?

Reply:
`@RayRemote` isn't needed now, because `CallOptions` is the preferred way to pass in options. I'm thinking of deprecating `@RayRemote`.